src/backend/aspen/app/views/phylo_runs.py

```diff
@@ -143,5 +147,41 @@ def start_phylo_run():
 
     # TODO - invoke a step function!
     # boto3.invok_step_something(parameters)
+    aspen_config = application.aspen_config
+
+    sfn_input_json = {
+      "Input": {
+        "Run": {
+          "aspen_config_secret_name": os.environ.get("ASPEN_CONFIG_SECRET_NAME", "aspen-config"),
+          "aws_region": aws.region(),
+          "docker_image_id": aspen_config.NEXTSTRAIN_DOCKER_IMAGE_ID,
+          "group_name": group.name,
+          "remote_dev_prefix": "",
+          "s3_filestem": f"{group.location} {request_data["tree_type"].capitalize()}",
+          "workflow_id": workflow.id,
+        },
+      },
+      "OutputPrefix": aspen_config.NEXTSTRAIN_OUTPUT_PREFIX,
+      "RUN_WDL_URI": aspen_config.RUN_WDL_URI,
+      "RunEC2Memory": aspen_config.NEXTSTRAIN_EC2_MEMORY,
+      "RunEC2Vcpu": aspen_config.NEXTSTRAIN_EC2_VCPU,
+      "RunSPOTMemory": aspen_config.NEXTSTRAIN_SPOT_MEMORY,
+      "RunSPOTVcpu": aspen_config.NEXTSTRAIN_SPOT_VCPU,
+    }
+
+    session = aws.session()
+    client = session.client(
+        service_name="stepfunctions",
+        endpoint_url=os.getenv("BOTO_ENDPOINT_URL") or None
+    )
+
+    try:
+        response = client.start_execution(
+            stateMachineArn=os.environ.get("NEXTSTRAIN_SFN_ARM"),
```

Comment:
This should be ARN

Reply:
oops got excited about ARM computing for a second there